app/components/agenda-manager/edit.js

```diff
@@ -5,15 +5,33 @@ import { inject as service } from '@ember/service';
 
 export default class AgendaManagerEditComponent extends Component {
   @service documentService;
-
+  oldGeplandOpenbaar;
```

Comment:
I think this can just be a getter and then the did-update can be dropped altogether